main.tf

```diff
@@ -14,7 +14,7 @@ locals {
 }
 
 resource "azurerm_application_gateway" "ag" {
-  name                = "${local.resource_prefix}aks-fe-${format("%02d", count.index)}-${var.env}-agw"
+  name                = var.enable_waf ? "${local.resource_prefix}aks-fe-${format("%02d", count.index)}-${var.env}-agw-waf" : "${local.resource_prefix}aks-fe-${format("%02d", count.index)}-${var.env}-agw"
```

Comment:
Should we remove `aks` from the name of the WAF AppGW? It's not in the aks appgw subnet and could be used for any purpose unrelated to AKS

Reply:
sounds like a good idea, i'll remove that now